### ISSUE TITLE
Make links look like links

### ISSUE THREAD

**Issue by Tjoppen** [arbeitszeit/arbeitszeitapp#944]
Hi

When looking at the dashboard the links "Private consumption", "My account", "My consumptions" etc do not look like links. This is dubious from an a11y standpoint. It also seems \<h1> is being abused for sizing the link "buttons", when the semantically more correct way should be a unordered list of links below the "My area" heading.

**Comment by sloschert** [comment]
Hi @Tjoppen, I would not agree that it's just a time bank. If you are interested and haven't done it yet you can read more on the underlying theory in [this book](https://arbeitszeit.noblogs.org/files/2023/04/GIC-Fundamental-Principles-2.-Ed.1935-1.pdf).

**Comment by Tjoppen** [comment]
I am aware of the work of the GIC. I was actually sent that exact book for review some time ago, but have not gotten around to it. I have absorbed some of it through Tom O'Brien's podcast.

As far as I can tell at the moment, there is nothing in the present implementation that can be used for operational in-kind planning. Quite the opposite: all in-kind information is as far as I can tell tossed away in favor of labour time accounting. This is an extreme form of aggregation, far worse than that used in the USSR under Krushchev. Anarchy in production is likely to result. Compare Marx pointing out in vol III that accounting must expand in scope and detail in communism. I am aware that some modern followers of the GIC reject this notion.

**Comment by sloschert** [comment]
“All in-kind information is tossed away” is not accurate IMO. Companies and society get useful information in a distributed manner that can and will be used for in-kind planning as well.

To start with: Companies file plans for each of the products or services they offer. Each plan contains the product definition and the unit of distribution. A plan can be associated with plans of the same product of other companies in a „plan cooperation“ (which is a way of individual companies to agree that they produce the same product).

You have then detailed public information on how many units of a certain product have been consumed productively at what time by which company. In the same way there is public information on (anonymized) private consumption.

**Comment by Tjoppen** [comment]
I don't see anything like this in the current implementation, unless I'm missing something. How for example is the gas composition of the atmosphere to be constrained globally?
